.github/workflows/ci.yml

```diff
@@ -7,10 +7,13 @@ jobs:
     runs-on: ubuntu-latest
     
     steps:
-    - uses: actions/checkout@v2
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Check Composer configuration
+      run: composer validate --strict
     - name: Install dependencies
       run: composer install
     - name: Start Redis
       uses: supercharge/redis-github-action@1.1.0
-    - name: Run tests
-      run: phpunit
\ No newline at end of file
+    - name: Run unit tests
```

Comment:
They're not exactly unit tests. They for example execute the queue worker to run the queued jobs.

Revert this one change and I'll merge.

Thanks.

Reply:
PHPUnit running non-unit tests??